diff --git a/android/app/build.gradle b/android/app/build.gradle
index c52d428..5ea29fe 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -6,8 +6,8 @@ plugins {
 
 android {
     namespace = "syncara.app"
-    compileSdk = flutter.compileSdkVersion
-    ndkVersion = flutter.ndkVersion
+    compileSdk = 36
+    ndkVersion = "27.0.12077973"
 
     compileOptions {
         sourceCompatibility = JavaVersion.VERSION_17
@@ -29,7 +29,7 @@ android {
     defaultConfig {
         applicationId = "syncara.app"
 
-        minSdk = flutter.minSdkVersion
+        minSdk = 23
         targetSdk = flutter.targetSdkVersion
 
         versionCode = flutter.versionCode
diff --git a/pubspec.yaml b/pubspec.yaml
index 6f3aea8..0d31044 100644
--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -19,8 +19,8 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 version: 0.1.9+10
 
 environment:
-  sdk: ^3.10.4
-  flutter: 3.38.5
+  sdk: ^3.8.1
+  flutter: 3.32.8
 
 # Dependencies specify other packages that your package needs in order to work.
 # To automatically upgrade your package dependencies to the latest versions
